codebase/superdarn/src.bin/tk/plot/time_plot.1.8/time_plot.c

```diff
@@ -1099,9 +1104,11 @@ int main(int argc,char *argv[]) {
                                prm->time.sc+prm->time.us/1.0e6);
        TimeEpochToYMDHMS(stime,&yr,&mo,&dy,&hr,&mt,&sc);
        status=OldFitSeek(oldfitfp,yr,mo,dy,hr,mt,0,NULL);
+       status=OldFitRead(oldfitfp,prm,fit);
```

Comment:
Why is this needed here?

Reply:
`FitFSeek` and `OldFitSeek` work in different ways. Without the additional `OldFitRead` line, the following code will produce a blank plot (no data and no radar information):
```time_plot -x -a -old [file]```

This problem only manifests itself when plotting all beams, so ```time_plot -x -a -old -b 7 [file]``` will work just fine. I'm not sure exactly why this is, so there might be a more elegant solution. But the OldFitRead line is not redundant.

Reply:
Thanks! This code is really not easy to read, so I thought I would ask.

Sounds like a different issue that can be addressed by a different pull request if needed.